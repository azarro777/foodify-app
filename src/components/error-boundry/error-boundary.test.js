import React from 'react';
import ErrorBoundry from './error-boundry';
import { shallow } from 'enzyme';

describe('Testing <ErrorBoundry/> component', () => {
  const errorBoundary = shallow(<ErrorBoundry />);
  it('ErrorBoundry have rendered correctly', () => {
    expect(errorBoundary).toMatchSnapshot();
  });
  it('ErrorBoundary state is folse', () => {
    expect(errorBoundary.state().error).toBeFalse();
  });
});
