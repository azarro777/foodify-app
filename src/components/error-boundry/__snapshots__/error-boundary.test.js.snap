// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <ErrorBoundry/> component ErrorBoundry have rendered correctly 1`] = `ShallowWrapper {}`;
